docs/CAPS/Specifications.md

```diff
@@ -1,7 +1,96 @@
+This document present the specifications of the CAPS format version `1.0.0`.
+
+## Versions of Clinica, BIDS, and CAPS specifications
+
+| Clinica version | BIDS version supported | CAPS version supported |
+|-----------------| ---------------------- | ---------------------- |
+| 0.9.x           |         1.7.0          |          1.0.0         |
+| 0.8.x           |         1.7.0          |       no version       |
+
+## The dataset_description.json file
+
+### Specifications
+
+This file MUST be present at the root of a CAPS dataset and MUST contain the following minimal information:
+
+```json
+{
+	"Name": "name identifier for the dataset",
+	"BIDSVersion": "1.7.0",
+	"CAPSVersion": "1.0.0",
+	"DatasetType": "derivative"
+}
+```
+
+- `Name`: String identifier of the dataset. It can be the name of your study for example. By default Clinica generates a random UUID for this field.
+- `BidsVersion`: The version number of the BIDS specifications that the BIDS input dataset is using when this CAPS dataset was generated.
+- `CAPSVersion`: The version number of the CAPS specifications used for this dataset.
+- `DatasetType`: Either "raw" or "derivative". For a CAPS dataset this should always be "derivative" as it contains processed data.
+
+In addition, the `dataset_description.json` file MAY contain a `Processing` key which is a list of objects describing the different processing pipelines that were run on this CAPS.
+Here is an example for a CAPS dataset containing the outputs of two pipelines: `t1-linear` and `pet-linear`:
+
+```json
+{
+    "Name": "e6719ef6-2411-4ad2-8abd-da1fd8fbdf32",
+    "BIDSVersion": "1.7.0",
+    "CAPSVersion": "1.0.0",
+    "DatasetType": "derivative",
+    "Processing": [
+        {
+            "Name": "t1-linear",
+            "Date": "2024-08-06T10:28:21.848950",
+            "Author": "ci",
+            "Machine": "ubuntu",
+            "ProcessingPath": "subjects/*/*/t1_linear",
+            "InputPath": "/mnt/data_ci/T1Linear/in/bids"
+        },
+        {
+            "Name": "pet-linear",
+            "Date": "2024-08-06T10:36:27.403373",
+            "Author": "ci",
+            "Machine": "ubuntu",
+            "ProcessingPath": "subjects/*/*/pet_linear",
+            "InputPath": "/mnt/data_ci/PETLinear/in/bids"
+        }
+    ]
+}
+```
+
+A `Processing` is described with the following fields:
+
+- `Name`: The name of the processing. For Clinica pipelines, this is the name of the pipeline.
+- `Date`: This date is in iso-format and indicates when the processing was run.
+- `Author`: This indicates the user name which triggered the processing.
+- `Machine`: This indicates the name of the machine on which the processing was run.
+- `ProcessingPath`: This is a path regex (relative to the CAPS dataset root) that can be used to get all sub-folders having data for this processing.
```

Comment:
That is not exactly true in case you have several processings with different names but similar pipelines, right ?
For example : if I choose to run T1-linear twice on different BIDS subjects going into the same CAPS output, and naming the processing two different names.

Reply:
(I may be missing something here)